src/lib/containers/CardContainerLogic.tsx

```diff
@@ -32,9 +32,11 @@ export type MarkdownLink = {
   matchColumnName: string
 }
 
-export type CTAButtonCardLink = {
-  buttonText: string
-  linkConfig: CardLink
+export type CTACardLink = {
```

Comment:
Remove "Button" from name, since it's no longer a button.  Simplify so that it only supports link text and a table/view column that contains the URLs (or Synapse IDs!)